### PR TITLE
Add sample for load data to data warehouse

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: load-data-to-bigquery/main.bal [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import ballerinax/googleapis.bigquery;
+import ballerina/uuid;
+
```
Comment (ShammiL) [inline load-data-to-bigquery/main.bal]:
```suggestion
import ballerina/io;
import ballerina/uuid;
import ballerinax/googleapis.bigquery;
```

File: load-data-to-bigquery/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[package]
+org = "lakshanweerasinghe"
```
Comment (SasinduDilshara) [inline load-data-to-bigquery/Ballerina.toml]: shall we add a `org` name like `ballerina`.

File: load-data-to-bigquery/main.bal [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    string Converted;
+    string ConversionDate;
+    float OpportunityAmount;
+    string OpportunityStage;
+|};
+
+configurable string bigQueryAccessToken = ?;
+configurable string projectId = ?;
+configurable string datasetId = ?;
+configurable string tableId = ?;
```
Comment (SasinduDilshara) [inline load-data-to-bigquery/main.bal:17]: shall we move the config variables to the top

File: load-data-to-bigquery/.devcontainer.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (pcnfernando) [inline load-data-to-bigquery/.devcontainer.json]: Can remove this

File: load-data-to-bigquery/Ballerina.toml (hunk 2 above)
Comment (pcnfernando) [reply load-data-to-bigquery/Ballerina.toml]: use `etl`

File: load-data-to-bigquery/Ballerina.toml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+name = "load_data_to_bigquery"
+version = "0.1.0"
+distribution = "2201.8.2"
+
+[build-options]
+observabilityIncluded = true
```
Comment (pcnfernando) [inline load-data-to-bigquery/Ballerina.toml]: Can remove this

**Comment by pcnfernando** [comment]
Need to add Readme.md

File: load-data-to-bigquery/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+interactionId   | STRING | REQUIRED
+interactionType | STRING | REQUIRED
+timestamp       | STRING | REQUIRED
+platform        | STRING | REQUIRED
+message         | STRING | REQUIRED
+```
+
+### Updating the Config.toml file
+```
+bigQueryAccessToken = "<BIG_QUERY_ACCESS_TOKEN>"
```
Comment (pcnfernando) [inline load-data-to-bigquery/README.md:20]: This doens't mention where the Config.toml should go etc

File: load-data-to-bigquery/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+## Steps to run the sample
+
+### Configuring Big Query
```
Comment (pcnfernando) [inline load-data-to-bigquery/README.md]: Shall we rephrase these as steps?
1. Configure Big Query Data warehouse
2. Configure Config.toml
3.....

Comment (LakshanWeerasinghe) [reply load-data-to-bigquery/README.md]: Updated in [28a3a](https://github.com/ballerina-guides/etl-samples/pull/6/commits/28a3a442c39b15722882767eca520e6a04bb269e).

File: load-data-to-bigquery/README.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+bigQueryAccessToken = "<BIG_QUERY_ACCESS_TOKEN>"
+projectId = "<PROJECT_ID>"
+datasetId = "<DATASET_ID>"
+tableId = "<TABLE_ID>"
+```
+2. Obtain a [Google API key](https://developers.google.com/identity/protocols/oauth2) and replace with `<BIG_QUERY_ACCESS_TOKEN>`.
+3. Find the `projectId` from the big query warehouse project and replace with `<PROJECT_ID>`.
+4. Find the `datasetId` from the project and replace with `<DATASET_ID>`.
+5. FInd the `tableId` from the dataset and replace with `<TABLE_ID>`.
+
```
Comment (pcnfernando) [inline load-data-to-bigquery/README.md:29]:
```suggestion
```

**Review by pcnfernando — APPROVED** [review]

**Pull request merged**